tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+
+# cert-manager install process adapted from:
+# https://docs.cert-manager.io/en/latest/getting-started/install/kubernetes.html#installing-with-regular-manifests
```

Comment:
This link appears to have changed slightly, to
`https://cert-manager.io/docs/installation/kubernetes/#installing-with-regular-manifests`